pythermalcomfort/models/phs.py

```diff
@@ -357,30 +391,35 @@ def _phs_optimized(
         def_speed = 1
 
     # radiating area dubois
-    a_r_du = 0.7
     if posture == Postures.standing.value:
         a_r_du = 0.77
-    if posture == Postures.crouching.value:
+    elif posture == Postures.sitting.value:
```

Comment:
This change will introduce a warning in PyCharm since the variable may not be defined. Perhaps the previous code was better.